src/citation/views/citation_entry_view.py

```diff
@@ -357,6 +368,19 @@ def upload_library(self, request):
         )
 
     @action(detail=False, methods=["get"], permission_classes=[IsAuthenticated])
+    def saved_org_citations(self, request):
+        """Returns all citations saved within the user's organization(s)"""
+
+        user = User.objects.get(id=request.user.id)
+        organizations = get_user_organizations(user)
+        saved_citations = self.get_queryset().filter(
+            organization_id__in=[o.id for o in organizations]
+        )
+
+        serializer = MinimalCitationEntrySerializer(saved_citations, many=True)
+        return Response(serializer.data)
+
+    @action(detail=False, methods=["get"], permission_classes=[AllowAny])
```

Comment:
Did the permission_classes on user_citations change?

Reply:
Good catch, I was testing and forgot to revert back